docs/MapColonies/vector/Services/feedback-api/README.md

```diff
@@ -0,0 +1,144 @@
+                        [ -118.30879709771484, 33.71635922964194 ],
+                        [ -118.30619642115158, 33.71550819588987 ],
+                        [ -118.30586490633668, 33.715921827872904 ],
+                        [ -118.30587062210924, 33.716183318328746 ],
+                        [ -118.30812263653988, 33.71684417247593 ]
+                    ]
+                ],
+                "type": "Polygon"
+            },
+            "properties": {
+                "score": 1.2880917,
+                "matches": [ { "layer": "osm_schools", "source": "OSM", "source_id": [ "1a5b981b-bb0e-44dd-b9e2-424b92f2de49" ] } ],
+                "names": { "en": [ "White Point Elementary School" ], "fr": [ "Escuela Primaria White Point" ], "default": [ "White Point Elementary School" ], "display": "White Point Elementary School" },
+                "placetype": "education",
+                "sub_placetype": "school",
+                "regions": [ { "region": "USA", "sub_region_names": [ "Los Angeles" ] } ]
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "coordinates": [ [
+                        [ 2.346441270696971, 48.88088750665477 ],
+                        [ 2.3462780852304945, 48.88018258877358 ],
+                        [ 2.347503576087604, 48.87999951892243 ],
+                        [ 2.347737155284733, 48.88070864783427 ],
+                        [ 2.346441270696971, 48.88088750665477 ] ] ],
+                "type": "Polygon"
+            },
+            "properties": {
+                "score": 1.2880917,
+                "matches": [
+                    { "layer": "osm_schools", "source": "OSM", "source_id": [ "dc02a3f9-156a-4f61-85bd-fd040cd322a3" ] } ],
+                "names": { "en": [ "Wi School Paris 9" ], "fr": [ "Ecole Wi Paris 9" ], "default": [ "Wi School Paris 9" ], "display": "Wi School Paris 9" },
+                "placetype": "education",
+                "sub_placetype": "school",
+                "regions": [ { "region": "FRANCE", "sub_region_names": [ "Paris" ] } ]
+            }
+        }
+    ]
+}
+```
+</details>
+
+Notice this response listed 2 features.<br/><br/>
+When sending the response to the `feedback API`, you need to provide two parameters:<br/>
+📍 Request ID - from the header `'x-request-id'`<br/>
+📍 Chosen Response ID - the ID of the user's selected response (corresponding index of the result). <br/><br/>
+
+Lets think of the current response as the actual response in this case:<br/>
+📍 If the user chose `"White Point Elementary School"`, the Chosen Response ID would be `0`.<br/>
+📍 If the user chose `"Wi School Paris 9"`, the Chosen Response ID would be `1`.<br/><br/>
+
+Then our request to the `feedback API` would look like this: <br/>
+
+```curl title="Geocoding's Feedback Api Request"
+curl --location '<feedback_api_url>/feedback' \
+--header 'x-api-key: <x-api-key>' \
+--header 'x-user-id: <x-user-id>'
+```
+
```

Comment:
This is not the right curl. 
First, it isn't POST request. missing body payload